create_database.py

```diff
@@ -0,0 +1,136 @@
+"""Read in data from a text file. Manipulate the data and insert it
+into an SQLite database with a 'users table' that has columns:
+userID, given_name, family_name
+and a 'cars table' that has columns:
+car, owner
+Return a completion message upon completion.
+"""
+import sqlite3
+import name
+import car
+import read_file
+import user_id
+
+def unpack_data(data_package):
+    """Take in a list of 3 strings which contain data in the following
+    formata:
+    ['f: <given_name>',
+     'l: <family_name>',
+     'c: <car>, <car>, <car>,...']
+    If the data package contains valid examples of all three elements,
+    extract the elements from their respective labels and return a
+    tuple of the isolated elements. Otherwise, return None.
+    """
+    assert len(data_package) == 3, "Data packages consist of 3 strings of information."
+    given_name = name.get_given_name_from_label(data_package[0].lower())
+    family_name = name.get_family_name_from_label(data_package[1].lower())
+    user_cars = car.get_from_label(data_package[2])
+
+    if given_name and family_name and user_cars:
```

Comment:
i don't see any use of your own objects throughout - even if you have to create objects that are just getters and setters, the capstone is a good place to show you know how to create an object/class if you need to. i'd suggest grouping given name, family name and user cars into one object rather than passing around a tuple.